services/user-svc/internal/models/user_models.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+import "github.com/google/uuid"
+
+type BaseUser struct {
+	Email    string `gorm:"column:email"`
+	Nickname string `gorm:"column:nickname"`
+	Name     string `gorm:"column:name"`
```

Comment:
Just another spontanious idea: Add `AvatarUrl` here.
On create generate a sha-hash of the Email.
Insert `https://source.boringavatars.com/marble/128/{hash}` into the database (this is always the same picture).

I feel like this is really easy to implement and will save us a lot of time later on.